README.md

```diff
@@ -67,6 +81,22 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
 	- [Russian](#russian)
 
 
+Icons and tags:
+
+- Ancient: ![ancient]
+- Commercial: ![$]
+- Crush: ![heart]
+- Supporting: ![support]
+- Up and coming: ![upcoming]
+- Web Based: ![web]
+- Beta: ![beta]
+- Video: ![video]
+- Open Source: ![oss]
```

Comment:
isn't it easier to just list the closed source projects  since open source (project on github) is actually the default here ?